ADALiOS/ADALiOS/ADAL.h

```diff
@@ -19,12 +19,10 @@
 
 #import "ADLogger.h"
 #import "ADAuthenticationContext.h"
-#import "ADTokenCacheStoring.h"
 #import "ADAuthenticationError.h"
 #import "ADAuthenticationResult.h"
-#import "ADTokenCacheStoreItem.h"
+#import "ADTokenCacheItem.h"
 #import "ADUserInformation.h"
-#import "ADTokenCacheStoreKey.h"
```

Comment:
What's it being used for though. The public API on the cache now is:

allItems and removeItem:, taking in a ADTokenCachItem, this matches the API surface on .NET. There is no public usage of ADTokenCacheKey w.r.t. caching anymore, as such it becomes an internal implementation detail and shouldn't be made public